react/components/TrackEventsWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+import { forEach } from 'ramda'
+import { PureComponent } from 'react'
+import ReactDOM from 'react-dom'
+
+interface Props {
+  events?: string[]
+  id?: string
+}
+
+const sendEvent = (id: string, event: string) => {
+  window.postMessage({
+    pageComponentInteraction : {
+      blockId: id,
+      interactionType: event,
+      namespace: 'render-runtime',
+    }
+  }, '*')
+}
+
+class TrackEventsWrapper extends PureComponent<Props> {
+
+  public eventsHandler: Record<string, () => void> = {}
+
+  public componentDidMount(){
+    const { id, events } = this.props
+    if(events && events.length && id){
+      const element = ReactDOM.findDOMNode(this)
```

Comment:
can we find another way around this without the use of `findDOMNode`? This method is getting deprecated and have several issues, specially regarding component update.

Reply:
I searched, but didn't find anything that could replace it in this case

Reply:
It's already being used in https://github.com/vtex-apps/render-runtime/blob/master/react/components/ExtensionPoint.tsx#L176